Userland/Utilities/groupdel.cpp

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2020, Fei Wu <f.eiwu@yahoo.com>
+ * Copyright (c) 2021, Brandon Pruitt <brapru@pm.me>
+ * Copyright (c) 2021, Maxime Friess <M4x1me@pm.me>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/ScopeGuard.h>
+#include <AK/String.h>
+#include <AK/StringBuilder.h>
+#include <LibCore/Account.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <ctype.h>
+#include <dirent.h>
+#include <errno.h>
+#include <grp.h>
+#include <pwd.h>
+#include <shadow.h>
+#include <spawn.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
+
+int main(int argc, char** argv)
+{
+    if (pledge("stdio wpath rpath cpath fattr proc exec", nullptr) < 0) {
+        perror("pledge");
+        return 1;
+    }
+
+    if (unveil("/etc/", "rwc") < 0) {
+        perror("unveil");
+        return 1;
+    }
+
+    if (unveil("/bin/rm", "x") < 0) {
+        perror("unveil");
+        return 1;
+    }
+
+    const char* groupname = nullptr;
```

Comment:
Use east const style: `char const*`. This comment applies to the diff as a whole.